Makefile

```diff
@@ -0,0 +1,7 @@
+ZOOKEEPRER_VERSION ?= 3.7.0
```

Comment:
please add this information in the Readme

Reply:
What is the purpose?

Reply:
For version lower than this, it may not work, so it's important to make it clear for users

Reply:
Hi, I have tested 3.4.0 - 3.7.0 (exclude beta). all passed the unit test.